datadog-siem-cloud/info.json

```diff
@@ -0,0 +1,1536 @@
+            "time_to_detect": "",
+            "time_to_repair": "",
+            "created_by_uuid": "",
+            "field_analytics": "",
+            "time_to_resolve": "",
+            "customer_impacted": "",
+            "incident_type_uuid": "",
+            "customer_impact_end": "",
+            "non_datadog_creator": "",
+            "notification_handles": [],
+            "customer_impact_scope": "",
+            "customer_impact_start": "",
+            "last_modified_by_uuid": "",
+            "creation_idempotency_key": "",
+            "customer_impact_duration": "",
+            "time_to_internal_response": ""
+          },
+          "relationships": {
+            "impacts": {
+              "data": []
+            },
+            "responders": {
+              "data": [
+                {
+                  "id": "",
+                  "type": ""
+                }
+              ]
+            },
+            "attachments": {
+              "data": []
+            },
+            "integrations": {
+              "data": []
+            },
+            "commander_user": {
+              "data": {
+                "id": "",
+                "type": ""
+              }
+            },
+            "created_by_user": {
+              "data": {
+                "id": "",
+                "type": ""
+              }
+            },
+            "user_defined_fields": {
+              "data": [
+                {
+                  "id": "",
+                  "type": ""
+                }
+              ]
+            },
+            "last_modified_by_user": {
+              "data": {
+                "id": "",
+                "type": ""
+              }
+            }
+          }
+        },
+        "included": []
+      }
+    },
+    {
+      "operation": "search_incidents",
+      "annotation": "search_incidents",
+      "title": "Search Incidents",
+      "description": "Search and retrieve the detailed list of incidents based on the parameters that you have specified.",
+      "category": "investigation",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "State",
+          "name": "state",
+          "type": "select",
+          "description": "Specify the state of the incident to filter the results.",
+          "tooltip": "Specify the state of the incident to filter the results.",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "options": [
+            "Active",
+            "Stable",
+            "Resolved"
+          ]
+        },
+        {
+          "title": "Severity",
+          "name": "severity",
+          "type": "select",
+          "description": "Specify the state of the incident to filter the results.",
+          "tooltip": "Specify the state of the incident to filter the results.",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "options": [
+            "SEV-1 (Critical)",
```

Comment:
To show user we can show only "Critical", "Hight" string in drop down, internally value can be SEV-1, SEV-2.

Reply:
I think we can keep it like this as in the Datadog platform it is displayed like this SEV-1, SEV-2. So it will be more clear for user to keep like this SEV-1 (Critical).